css/buttons.styl

```diff
@@ -44,6 +44,8 @@
 
 .pill-button
   @extends $reset-button
+  color: FOREGROUND !important
+  border-color: FOREGROUND !important
```

Comment:
The `pill-button` class is used around the whole site especially in the project builder. This is modifying this class across the board. Please don't add these here with `!important`. Instead write a new class scoped where it will be used and use `@extends` if you need to.